src/cluster/autoscale.go

```diff
@@ -391,3 +403,30 @@ func deployAutoscalerChart(cluster CommonCluster, nodeGroups []nodeGroup, kubeCo
 	log.Infof("'%s' %sed", chartName, action)
 	return nil
 }
+
+func getImageVersion(cluster CommonCluster) map[string]string {
+	k8sVersion := "1.12"
+	if c, ok := cluster.(interface{ GetKubernetesVersion() (string, error) }); ok {
+		k8sVersion, _ = c.GetKubernetesVersion()
+	}
+	for _, imageVersion := range global.Config.Cluster.Autoscale.Charts.ClusterAutoscaler.ImageVersions {
+		version, err := semver.NewVersion(k8sVersion)
```

Comment:
This doesn't need to be performed for every iteration.

Reply:
fixed